### ISSUE TITLE
Suzieq shows bgp sessions which has been removed from the device

### ISSUE THREAD

**Issue by rasanentimo** [netenglabs/suzieq#458]
**Describe the bug**
Suzieq shows bgp sessions which has been removed from the device

**To Reproduce**
1. create bgp configuration
2. poll the data with suzieq
3. delete bgp configration
4. the data is still shown
5. delete older parquet files
6. the data is still shown

I'm not sure if this requires some other steps like restarting the poller at some phase.

**Attach**
n/a

 **Expected behavior**
Expecting the bgp session not be visible in suzieq after it has been removed from device

Expecting the bgp session not be visible in suzieq after the older parquet files have been deleted. 

**Screenshots**
n/a

**Additional context**
This bgp session is not present on the device but it's shown on suzieq

```
         peer    state                  estdTime                        timestamp
0  Ethernet39  NotEstd 1970-01-01 00:00:00+00:00 2021-10-09 10:29:41.690000+00:00
```
the parquet files doesn't exist for that date either

```
# ls -lah
total 40M
drwxr-xr-x 2 root root  40M Oct 29 20:02 .
drwxr-xr-x 5 root root  83M Sep 23 11:00 ..
-rw-r--r-- 1 root root  92K Oct 10 00:01 sqc-h-1633820400-1633824000.parquet
-rw-r--r-- 1 root root 102K Oct 10 01:01 sqc-h-1633824000-1633827600.parquet
-rw-r--r-- 1 root root 102K Oct 10 02:01 sqc-h-1633827600-1633831200.parquet
-rw-r--r-- 1 root root 102K Oct 10 03:01 sqc-h-1633831200-1633834800.parquet
-rw-r--r-- 1 root root 102K Oct 10 04:01 sqc-h-1633834800-1633838400.parquet
-rw-r--r-- 1 root root  92K Oct 10 05:01 sqc-h-1633838400-1633842000.parquet
-rw-r--r-- 1 root root  92K Oct 10 06:01 sqc-h-1633842000-1633845600.parquet
-rw-r--r-- 1 root root  92K Oct 10 07:01 sqc-h-1633845600-1633849200.parquet
```

**Comment by ddutt** [comment]
The reason for this behavior is because some BGP configuration was removed from the device when the poller was not operational. In other words, there was a BGP session X, the poller was stopped, the session X removed from configuration, and the poller restarted. The restarted poller has no knowledge of session X being deleted, and so that old session info persists. The coalescer continues to sweep up the older parquet data into newly coalesced data and so the info about session X remains. 

If the session X had been removed from the configuration while the poller was running, this issue should not happen. 

Nevertheless, this is a very possible condition to find oneself in, and Suzieq must provide support for handling this case. 

The workaround for now is to use start-time parameter. Luckily, the CLI provides setting the start-time once and forgetting about it. Not so for the REST API.

**Comment by ddutt** [comment]
Fixed in 0.17.1 via various commits.
